src/routes/root.ts

```diff
@@ -7,6 +7,26 @@ import config from '../config'
 const logger = customLogger(path.basename(__filename))
 module.exports = app => {
 	//:TODO need to secure these apis too.
+	const cache = app.get('redisCache')
+	const db = app.get('db')
+	const { sequelize, Sequelize } = db
+
+	// Todo table
+	const Todo = sequelize.define('Todo', {
```

Comment:
Instead of defining model like this, this should be a separate file in a models folder. Sync can happen in application initialization code, like how other modules are initialized